src/main/java/frc/robot/Constants.java

```diff
@@ -4,35 +4,14 @@
 
 package frc.robot;
 
-import edu.wpi.first.math.trajectory.TrapezoidProfile;
-
-/**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
- * globally (i.e. public static). Do not put anything functional in this class.
- *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
- * constants are needed, to reduce verbosity.
- */
 public final class Constants {
-  public static class OperatorConstants {
-    public static final int kDriverControllerPort = 0;
-  }
-  public static final boolean DRIVETRAIN_DEBUG = false;
-    public static final boolean ARM_DEBUG = false;
-    public static final boolean EXTENDER_DEBUG = false;
-
-    public static final boolean GRIPPER_DEBUG = false;
-    public static final boolean PDB_DEBUG = false;
-    public static final boolean APRILTAG_DEBUG = false;
-
-    public static final boolean ENABLE_LOGGING = true;
+    public static class OperatorConstants {
+        public static final int kDriverControllerPort = 0;
```

Comment:
This is the pattern that comes with the WPILIB default project and isn't one we've adopted in the past (nested OperatorConstants inside Constants). We also don't use the variable style used in wpilib (k..). Can we just switch this to a constant?

Reply:
done